azure/resource-containers/pf-management-group/classes/pf-management-group-access.ps1

```diff
@@ -1,25 +0,0 @@
-Class PfManagementGroupAccess : PfAccess{
-    [PfManagementGroupDefinition]$ManagementGroupDefinition
```

Comment:
All access classes have been removed.  Access code has been moved to private functions